Source/loadsave.cpp

```diff
@@ -640,7 +640,7 @@ void LoadMonster(LoadHelper *file, Monster &monster)
 	file->Skip(1); // Alignment
 	monster.exp = file->NextLE<uint16_t>();
 
-	if ((monster.flags & MFLAG_GOLEM) != 0) // Don't skip for golems
+	if (monster.isPlayerOwned()) // Don't skip for golems

```

Comment:
maybe this should be checking if the monster is berserked (or both?)

Reply:
Your change here seams correct. Baserked monsters should get there values from the monster table, but monsters spawned by the player is based on the values that where calculated then.